editions/tw5.com/tiddlers/releasenotes/ReleaseTemplate.tid

```diff
@@ -1,3 +1,6 @@
+created: 20211117231152248
+modified: 20211117231156728
+tags: Releases
```

Comment:
An unfortunate side effect of applying this tag to this template is that the template will now show up in the table of contents. Presumably that holds for other template tiddlers, so it would be good to figure out a consistent approach.

Reply:
If I remember correctly how the mechanism works, any tiddler that does not match a path filter will stay in its original location. So one option for such tiddler is not to create a path filter.

Path filters make most sense where we expect there to be more tiddlers that will match that filter and thus get routed to the appropriate place.

Reply:
May be we can use `Release Template`

Reply:
> May be we can use `Release Template`

`Release Template` or `ReleaseTemplate` ?

Reply:
I'd vote "Release Template"